html/elements/summary.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "summary": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/summary",
+          "support": {
+            "webview_android": {
+              "version_added": "4"
+            },
+            "chrome": {
+              "version_added": "12"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
```

Comment:
Please add the same note as for the details element (under consideration)

Reply:
Fixed (and, meanwhile, MS moved it to In development, so I fixed this too)